<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lenovo" ignored="0" name="Default test" tests="3" failures="0" timestamp="2021-09-26T12:54:58 IST" time="0.738" errors="0">
  <testcase name="@BeforeTest configuration" time="0.078" classname="com.TestNG_Sep_Batch.Runner_1">
    <failure type="java.lang.NoClassDefFoundError" message="io/reactivex/rxjava3/core/Observer">
      <![CDATA[java.lang.NoClassDefFoundError: io/reactivex/rxjava3/core/Observer
at com.TestNG_Sep_Batch.Runner_1.configuration(Runner_1.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: io.reactivex.rxjava3.core.Observer
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 25 more
]]>
    </failure>
  </testcase> <!-- @BeforeTest configuration -->
  <testcase name="@AfterTest flush_1" time="-1.632641098355E9" classname="com.TestNG_Sep_Batch.Runner_1">
    <skipped/>
  </testcase> <!-- @AfterTest flush_1 -->
  <testcase name="launch_Url" time="0.0" classname="com.TestNG_Sep_Batch.Runner_1">
    <skipped/>
  </testcase> <!-- launch_Url -->
  <testcase name="username" time="0.0" classname="com.TestNG_Sep_Batch.Runner_1">
    <skipped/>
  </testcase> <!-- username -->
  <testcase name="login_Btn" time="0.0" classname="com.TestNG_Sep_Batch.Runner_1">
    <skipped/>
  </testcase> <!-- login_Btn -->
</testsuite> <!-- Default test -->
